src/FileCollection/FileParser.ts

```diff
@@ -130,10 +130,14 @@ class FileParser {
 			title: file.basename,
 			slug: `ob-${file.stat.ctime}`,
 			content: JSON.stringify(listOfSection),
-			source_type: "obsidian",
 			description: "Obsidian vault",
 			heading: listOfH1,
-			outgoing_links: outgoingLinks
+			outgoing_links: outgoingLinks,
```

Comment:
outgoing_links has been changed to internal_links .
 And the schema has changed form List<String> to List<InternalLinkObject>
 The Schema for InternalLinkObject is  
{
slug : <String>,
id : <String?>,
}

so, 
internal_links : [
   {
     slug : 'apple',
     id : null,
   },
  {
     slug : 'fruit',
     id : null,
   },
]